packages/medusa/src/migrations/1679950645254-product-domain-impoved-indexes.ts

```diff
@@ -10,7 +10,7 @@ export class productDomainImprovedIndexes1679950645254
       /* You can turn of this settings if you are in a context with lots of variants) set enable_nestloop to off; */
       
       DROP INDEX IF EXISTS "IDX_17a06d728e4cfbc5bd2ddb70af";
-      CREATE INDEX IF NOT EXISTS idx_money_amount_variant_id ON money_amount (variant_id);
+      CREATE INDEX IF NOT EXISTS idx_product_variant_money_amount_id ON money_amount (variant_id);
```

Comment:
todo: We should avoid changing existing migration and create a new migration to alter the name of the index